Source/WebCore/rendering/RenderAttachment.cpp

```diff
@@ -97,8 +97,18 @@ void RenderAttachment::layout()
         layoutShadowContent(newIntrinsicSize);
 }
 
-LayoutUnit RenderAttachment::baselinePosition(FontBaseline, bool, LineDirectionMode, LinePositionMode) const
+LayoutUnit RenderAttachment::baselinePosition(FontBaseline fontBaseline, bool firstLine, LineDirectionMode lineDirectionMode, LinePositionMode linePositionMode) const
 {
+    if (auto* baselineElement = attachmentElement().wideLayoutElementForBaseline()) {
+        bool unusedIsReplaced;
+        auto attachmentRect = attachmentElement().pixelSnappedRenderRect(&unusedIsReplaced);
+        auto baselineElementRect = baselineElement->pixelSnappedRenderRect(&unusedIsReplaced);
+        auto baselineElementTop = baselineElementRect.y() - attachmentRect.y();
+        if (auto* baselineElementRenderBox = baselineElement->renderBoxModelObject())
+            return LayoutUnit(baselineElementTop) + baselineElementRenderBox->baselinePosition(fontBaseline, firstLine, lineDirectionMode, linePositionMode);
+        return baselineElementTop + baselineElementRect.height();
```

Comment:
(sorry for the noise I guess I didn't approve my review on GitHub when I first tried to add the comments)

Reply:
Thanks for noticing. I've filed follow-up https://bugs.webkit.org/show_bug.cgi?id=259172 to look into it.
It would only be a 1px difference, so I'm not too worried about it; but of course I'd prefer this to be correct eventually. Feel free to take it if you can't bear the thought of this egregious error surviving for long! 😄 (Or comment in the bug or radar if you feel it's more important than I realize.)